docs/en/01_Working_with_projects/04_Customising_the_default_functionality.md

```diff
@@ -45,11 +45,14 @@ and have it apply to *all* page types, then your code might look something like
 	:::php
 	// File: mysite/code/Page.php
 	<?php
+	
+	use CWP\CWP\PageTypes\BasePage;
+
 	class Page extends BasePage {
```

Comment:
It'd be cool to follow PSR-2 for code examples, so drop `{` to a new line for class and function declarations.

We can also remove `:::php` code block declarations in favour of unindented triple backtick \`\`\`php sections like on GitHub